databricks/koalas/__init__.py

```diff
@@ -79,7 +79,7 @@ def assert_pyspark_version():
     logging.warning(
         "'PYARROW_IGNORE_TIMEZONE' environment variable was not set. It is required to "
         "set this environment variable to '1' in both driver and executor sides if you use "
-        "pyarrow>=2.0.0."
+        "pyarrow>=2.0.0. "
```

Comment:
Just wanted to make sure this is intentional.

Reply:
Yes, otherwise the message doesn't have an space between sentences.